### PR TITLE
Update defintion of test goals to new version.

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: ur_robot_driver/config/test_goal_publishers_config.yaml [hunk 1]
```diff
@@ -18,7 +22,7 @@ publisher_scaled_joint_trajectory_controller:
       - wrist_2_joint
       - wrist_3_joint
 
-    check_starting_point: true
+    check_starting_point: false
```
Comment (destogl) [inline ur_robot_driver/config/test_goal_publishers_config.yaml:25]:
```suggestion
    check_starting_point: true
```

File: ur_robot_driver/config/test_goal_publishers_config.yaml [hunk 2]
```diff
@@ -47,7 +55,7 @@ publisher_joint_trajectory_controller:
...
```
Comment (destogl) [inline ur_robot_driver/config/test_goal_publishers_config.yaml:58]:
```suggestion
    check_starting_point: true
```

**Comment by fmauch** [comment]
I assume, this is accompanying https://github.com/ros-controls/ros2_controllers/commit/b31635a514d75a3ded928479b1f60781ed3b7be3. This should only be affecting rolling, right?

**Review by fmauch — APPROVED** [review]
This looks fine to me, but I'll wat with merging until ros2_controllers v3.5.0 hits main.

**Comment by destogl** [comment]
Yes. We can backport it also. Old behavior is deprecated.

**Comment by fmauch** [comment]
@Mergifyio backbort humble

**Comment by fmauch** [comment]
@Mergifyio backport humble

**Pull request merged**